perf/benchmark/values.yaml

```diff
@@ -14,7 +14,7 @@ appresources1:
     memory: "1000Mi"
 
 fortioImage: fortio/fortio:latest_release
-curlImage: tutum/curl:trusty
+curlImage: cfmanteiga/alpine-bash-curl-jq:latest
```

Comment:
We have moved to another curl image in other places in istio: `curlimages/curl`. Not sure if you want to maintain consistency.

Reply:
curlimages/curl does not seem to have a bash shell?

Reply:
Agree that it doesn't have a shell and instead of infinity we are specifying 10 yrs in the other places `"bin/sleep","3650d"`

Reply:
The while true loop can does the same thing for infinity, but I changed to "bin/sleep","3650d" for consistency